tensorflow/python/framework/dtypes.py

```diff
@@ -646,6 +646,12 @@ def size(self):
 ])
 tf_export("QUANTIZED_DTYPES").export_constant(__name__, "QUANTIZED_DTYPES")
 
+_PYTHON_TO_TF = {
+    int: int64,
+    float: float64,
```

Comment:
I'm not sure about these mappings. If you make a `tf.constant([1, 2, 3])` then the resulting dtype is `tf.int32` (similarly `tf.constant([1.0, 2.0, 3.0])` returns a `tf.float32` tensor). Perhaps this mapping should be the same?